lightning/src/ln/channel.rs

```diff
@@ -695,6 +695,19 @@ pub(super) struct Channel<Signer: Sign> {
 
 	/// This channel's type, as negotiated during channel open
 	channel_type: ChannelTypeFeatures,
+
+	// Our counerparty can offer us SCID aliases which they will map to this channel when routing
+	// outbound payments. These can be used in invoice route hints to provide privacy of which
+	// on-chain transaction is ours.
+	// We only bother storing the most recent SCID alias at any time, though our counterparty has
+	// to store all of them.
```

Comment:
If the SCID alias goal is to mask the on-chain funding transaction, what's the interest to _rotate_ them ? I think one alias should be enough to achieve the purpose.